### DISCUSSION TITLE
I would like to know if it is possible to contact a user if he doesn't have contact information

### DISCUSSION THREAD

**Discussion by santiagobaronz** [community/community#42816] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello community!

I would like to change my github username but it's already taken. I noticed that person using the name I want is no longer using github anymore (due to his Contribution activity). But there's no contact information in order to contact with him and try to exchange our names.

Is there any way to contact with someone who doesnt have contact information on his profile?

**Comment by radar-blip** [comment]
Maybe opening an issue in one of his or her repositories.

**Reply by santiagobaronz** [reply]
He hasn't repositories :(

**Comment by DSDmark** [comment]
Hi @santiagobaronz,

it's a global issue. Except my first discussion, I created on that topic. Not sure. But, Their username, which is not active, may become available for someone else to claim. In this case, you may be able to claim the username yourself after a certain amount of time has passed. You could try to reach out to GitHub support to see if they can provide any assistance in getting in touch with the user.

> you can create a support ticket on GitHub. In your ticket, explain the situation and request that the username be released so that you can use it. Keep in mind that GitHub may not be able to release the username to you, as they do not have control over the actions of individual users.

**Reply by DSDmark** [reply]
@santiagobaronz, I guess, on that matter, no one except GitHub support, can't help you.
